File: code/C/atividades/media_estudante.c
#include <stdio.h>

int main(void) {
    float nota_um, nota_dois = 0;
    printf("Nota 1:");
    scanf("%f", &nota_um);

    printf("Nota 2:");
    scanf("%f", &nota_dois);

    float resultado = (nota_um + nota_dois) / 2;
    printf("A media entre %.1f e %.1f é igual a %.1f\n", nota_um, nota_dois, resultado);
}

File: code/Go/atividades/media_estudante.go
package main

import "fmt"

func main() {
	var nota_um, nota_dois float64 = 0, 0
	fmt.Println("Nota 1:")
	fmt.Scanf("%f", &nota_um)

	fmt.Println("Nota 2:")
	fmt.Scanf("%f", &nota_dois)

	resultado := (nota_um + nota_dois) / 2
	fmt.Printf("A media entre %.1f e %.1f é igual a %.1f\n", nota_um, nota_dois, resultado)
}
